### DISCUSSION TITLE
I haven't been rewarded for 11 days

### DISCUSSION THREAD

**Discussion by tracy32792** [Chia-Network/chia-blockchain#8074] [Support]
I have 17000 OG plots ，It has been abnormal since 1 month ago。

My network, computer system and Chia farming interface display are all farming, and the log display does not exceed 5 seconds。

**Comment by rckjhn** [comment]
Same here 15000 OG plots last reward 24 days ago. chia version 1.2.1.  Ran usual diagnostics and logs look normal.
Any ideas?

**Comment by LYXQQ** [comment]
I have 8000 OG plots and 5000 NFTs，received 2 XCH after more than 30 days with ETW 5 days，since 1.2.0.

**Reply by wjblanke** [reply]
Are you claiming the self pooling rewards? What happens when you join a pool?

**Comment by lumiamax** [comment]
22200 OG plsts,received xch half month ago,and there is lots of my friends （at least 2piB） didn't get coins more than 15days,so wahts going on there?

**Comment by rckjhn** [comment]
Collectively that's about 50k plots so ETW is about a day. This is good evidence for some sort of bug.

**Reply by psun888** [reply]
To confirm, I have around 58000 plots, 8 days without any reward. It seems the Chia network has a serious bug or under some kind of attacks to deny rewards to farmers. BTW, the farmer is running on a fastest super computer server and on a 200Gbps fiber connection port with one of world largest Internet providers,

**Reply by keliew** [reply]
if your drives are sleeping, the speed of your internet connection and server makes little to no difference.

**Reply by psun888** [reply]
Each harvester is configured with enterprise Raid system, and the OS is server system, there is no way hard drives sleeping. Plus, all the logs indicate very smooth farming condition. The situation only happens recently.

**Reply by keliew** [reply]
Very smooth farming condition sounds like there's absolutely no issues, which isn't the case.

Did you turn on `DEBUG` to check more thoroughly?

Probably a good idea to check sleep mode (at the firmware level) rather than to bat it off as a non-issue.

**Reply by jeansalim** [reply]
I haven't won for a week:
Plot count for all harvesters: 50763
Total size of plots: 4.907 PiB
Expected time to win: 1 day and 9 hours

It really seems like there's something wrong lately @psun888

**Comment by psun888** [comment]
One additional bug, The Chia farmer GUI is just unbelievable slow to open Plots, Farm and Pool tabs when you have more than 15K plots. At the moment, I have 58K plots, it takes about 30 minutes to open Farm tab, and 45 minutes to open Pool tab. I myself is a programmer, it's just too irresponsible to release such bugged software. I think Chia coin pricing will go down to less than one dollar just like original Burst coin. After all, Chia is an imitation of Burst anyway.

**Reply by keliew** [reply]
So there's a PR out there that you can try and see if it improves the performance of your setup.

**Comment by peterMa2009** [comment]
i have 26K plots,last time i got reward is Auguest 8th.I have check the log,but found nothing

**Reply by max4711qq** [reply]
2 wins in 35 days with a time to win of <2 days (about 1 day 15-20 hours ca.)
my logs are OK. My system is OK. This must be the network. I know many people with the same problem. We are not getting what the network and maths are promising us.

I hope this will be made public before they continue with their IPO. Then it will also hurt the founders, not just the farmers who got tricked into this. I have pulled the plug this morning and I am wiping clean my 3.6 TB as I am typing this.

I don't care about the lost money. But i am seriously annoyed about the waste of time I've lost since May.

**Comment by peterMa2009** [comment]
i hope somebody who developed chia blockchain can hear our sound. Thousand of farmers is the base,they are very important,don't hurt their heart!

**Comment by tracy32792** [comment]
I'm OGplots self farming,I haven't been rewarded for 20days发自我的华为手机-------- 原始邮件 --------发件人： wjblanke ***@***.***>日期： 2021年8月23日周一 凌晨2:11收件人： Chia-Network/chia-blockchain ***@***.***>抄送： tracy32792 ***@***.***>, Author ***@***.***>主    题： Re: [Chia-Network/chia-blockchain] I haven't been rewarded for 11 days (#8074)
Are you claiming the self pooling rewards? What happens when you join a pool?

**Comment by BrandtH22** [comment]
@tracy32792 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note - it is not uncommon to go 6x the ETW or more as wins are strictly based on probability and the time to win is not deterministic. It can also be the case that a farm will go 1/6th the ETW.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
